# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nvu/startup-script.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We must not use MOZILLA_FIVE_HOME ... if we want a working nvu ...

  - Rene Rebe <rene@exactcode.de>

--- mozilla/composer/app/mozilla.in.vanilla	2005-03-23 10:25:27.000000000 +0100
+++ mozilla/composer/app/mozilla.in	2005-03-23 10:25:52.000000000 +0100
@@ -36,9 +36,7 @@
 MRE_HOME=//opt/nvu/lib/mre/mre-0.90
 
 # honor MOZILLA_FIVE_HOME if it's there
-if [ -n "$MOZILLA_FIVE_HOME" ] ; then
-  dist_bin="$MOZILLA_FIVE_HOME"
-else
+
   # Use run-mozilla.sh in the current dir if it exists
   # If not, then start resolving symlinks until we find run-mozilla.sh
   found=0
@@ -77,7 +75,6 @@
       exit 1
     fi
   fi
-fi
 
 script_args=""
 moreargs=""
